src/App.scss

```diff
@@ -3,5 +3,17 @@
   margin: 0 auto;
   padding: 2rem;
   text-align: center;
+} 
+
+nav {
+  a{
+    margin-right: 10px;
+    text-decoration: none;
+    font-weight:700;
+    color:grey;
+  }
 }
 
+div{
+  color:grey
```

Comment:
is this intentional - you are setting grey color to ALL divs? If you want to have all text grey by default, then you can have the color: grey in #root , no need to have the extra div there

Reply:
Will create a new branch with styles and add all colors to variables.